src/zimscraperlib/zim/filesystem.py

```diff
@@ -134,7 +134,10 @@ def make_zim_file(
     redirects: Sequence[tuple[str, str, str]] = None,  # noqa: RUF013  # pyright: ignore
     redirects_file: pathlib.Path = None,  # noqa: RUF013  # pyright: ignore
     rewrite_links: bool = True,  # noqa: FBT001, FBT002, ARG001
-    workaround_nocancel: bool = True,  # noqa: FBT001, FBT002
+    workaround_nocancel: Optional[bool] = True,  # noqa: FBT002
+    compression: Optional[str] = None,
```

Comment:
This is not passed on purpose. We don't want to create ZIMs with anything but the default (zstd). libzim being a wrapper, it exposes the API has it can support multiple compression (although I'm not sure if we still include the required dependencies in the binary)

Reply:
Removed